packages/remote-config/android/src/main/java/io/capawesome/capacitorjs/plugins/firebase/remoteconfig/FirebaseRemoteConfigPlugin.java

```diff
@@ -76,9 +79,12 @@ public void error(String message) {
     @PluginMethod
     public void fetchConfig(PluginCall call) {
         try {
-            int minimumFetchIntervalInSeconds = call.getInt("minimumFetchIntervalInSeconds", 43200);
```

Comment:
This code has a warning `Unboxing of 'call.getInt("minimumFetchIntervalInSeconds", 43200)' may produce 'NullPointerException' `.

So, I changed the code.

Reply:
Please use the same code as below:

```java
Integer minimumFetchIntervalInSeconds = call.getInt("minimumFetchIntervalInSeconds", DEFAULT_MINIMUM_FETCH_INTERVAL_IN_SECONDS);
```